### PR TITLE
Add panels tab

### LINKED ISSUE #385: Bring over operational panel

### PR DESCRIPTION
**Pull request by eugenesk24**
### Description
Adds a new tab for Panels with rename/clone/delete panel disabled.
Change name of Service Map to Application Composition Map.
Clears storage if user cancels creating application.
Update create tests.

### Issues Resolved
#385 
#389 

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: dashboards-observability/public/components/application_analytics/components/application.tsx [hunk 1]
```diff
@@ -192,6 +196,26 @@ export function Application(props: AppDetailProps) {
...
+      <CustomPanelView
+        panelId={''}
+        http={http}
+        pplService={pplService}
+        chrome={chrome}
+        parentBreadcrumb={[parentBreadcrumb]}
+        // App analytics will not be renaming/cloning/deleting panels
+        renameCustomPanel={() => undefined}
+        cloneCustomPanel={():Promise<string> => Promise.reject()}
+        deleteCustomPanel={():Promise<string> => Promise.reject()}
```
Comment (joshuali925) [inline dashboards-observability/public/components/application_analytics/components/application.tsx:210]: why clone/delete are returning `Promise.reject()` but rename is returning `undefined`?

Comment (eugenesk24) [reply dashboards-observability/public/components/application_analytics/components/application.tsx:210]: They seem to have different return types, clone/delete required a Promise to be returned but rename allowed undefined. I can change them all to be `Promise.reject()` if it's confusing. The buttons to call those functions will be hidden from the UI.

Comment (ps48) [reply dashboards-observability/public/components/application_analytics/components/application.tsx:210]: Are you planning to remove these buttons in panels or put some bool flags in CustomPanelsView component.

**Review by ps48 — APPROVED** [review]

File: dashboards-observability/public/components/application_analytics/components/application.tsx (hunk 1 above)
Comment (eugenesk24) [reply dashboards-observability/public/components/application_analytics/components/application.tsx:210]: The second option, I use the page prop to decide whether to show the buttons or not.

**Review by joshuali925 — APPROVED** [review]

**Pull request merged**